Readme.md

```diff
@@ -125,9 +125,22 @@ Custom HTML file:
 ```html
 $ echo \
   '<!doctype html>
-  <link rel="stylesheet" href="/style.css" />
-  <script src="/elm.js"></script>
-  ' > index.html
+<html>
```

Comment:
Leaving two spaces of indentation below the prompt even if you break the line is more aesthetic and readable to me. [fish](https://fishshell.com/) – a very user-friendly shell – does that automatically as well.

The example is not meant to be copy-pastable, but to be understandable at first glance.